CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
+# v0.23.0
+
+## Announcements
+
+- Re-licensing: license change from
+  [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) to
+  [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0) (see #997 for
+  details)
+- `sbi` is now affiliated with [`NumFOCUS`](https://numfocus.org) 🎉
+- New contributors 🎉: @anastasiakrouglova, @theogruner, @felixp8, @Matthijspals,
+  @jsvetter, @pfuhr, @turnmanh, @fariedabuzaid, @augustes, @zinastef, @Baschdl,
+  @danielmk, @lisahaxel, @janko-petkovic, @samadpls, @ThomasGesseyJonesPX, @schroedk
+
+## Major Changes and New Features
+
+- change `sbi` default parameters: `training_batch_size=200`, `num_chains=20` (#1221,
+  @janfb)
+- more flexible API and shape handling for density estimator classes (#952, #965, #979,
+  #1151) (@gmoss13, @tomMoral, @manualgloeckler)
+- vectorized sampling and log_prob for `(S)NPE` given batches of x (#1153)
+  (@manuelgloeckler, @deismic)
+- batched sampling for vectorized MCMC samplers (#1176, #1210) (@gmoss13, @janfb)
+- support @zuko as a backend for normalizing flows (#1088, #1116)
+  (@anastasiakrouglova)
+- local c2st metric (#1109) (@JuliaLinhart)
+- tarp coverage metric (#1106) (@psteinb)
+- add interface for @PyMC samplers (#1053) (@famura, @felixp8)
+- big refactoring of plotting utilities, new tutorial (#1084) (@Matthijspals)
+- flow matching density estimators (#1049) (@turnmanh, @fariedabuzaid, @janfb)
+- score matching density estimators (#1015) (@rdgao, @jsvetter, @pfuhr,
+  @manuelgloeckler, @deismic, @janfb)
+- ABC methods for trial-based data using statistical distances (#1104)
+  (@theogruner)
+- improved tutorials and website documentation (#1012, #1051, #1073) (@augustes,
+  @zinaStef, @lisahaxel, @psteinb)
+- improved website structure and contribution guides (#1019) (@tomMoral, @janfb)
+- support Apple MPS as gpu device (#912) (@janfb)
+- dev container for using `sbi` in codespaces on GitHub (#1070) (@turnmanh)
+- enable importance sampling for likelihood-based estimators (#1183) (@manuelgloeckler)
+- refactoring and unified shape handling for `RatioEstimator` (#1097) (@bkmi)
+- faster sbc and tarp calibration checks via batched sampling (#1196) (@janfb)
+- batched sampling and embedding net support for `MNLE` (#1203) (@janfb)
+- better plotting options for coverage plots (#1039, #1212) (@janfb)
+- drop support for python3.8 and torch1.12 (#1233)
+- refactor folder structure and naming of `neural_nets`, and all inferences classes
```

Comment:
I would prefer to have the renaming of inference classes as one of the first items in this list

Reply:
I added the following to the first tutorial, maybe sth similar works here as well:

We renamed all inference classes from, e.g., `SNPE`, to `NPE` (i.e., we removed the `S` prefix). The functionality of the classes remains the same. The `NPE` class handles both the amortized (as shown in this tutorial) and sequential (as shown [here](https://sbi-dev.github.io/sbi/tutorial/02_multiround_inference/)) versions of neural posterior estimation.